src/index.js

```diff
@@ -25,8 +25,10 @@ export default (block, options = {}) => (props) => {
     .filter(a => a)
     .reduce((a, b) => a.concat(b), [])
 
-  const classNames = makers.map(maker => maker(blockName, props, { delimiters }))
+  const classNamesList = makers.map(maker => maker(blockName, props, { delimiters }))
+  const className = cx(classNamesList).trim().replace(/\s\s+/g, ' ')
```

Comment:
your're right.

another solution is to use: `replace(/\s{2,}/g, ' ')` (which means 2 or more)